modules/configuring-a-proxy-during-installation-ocm.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: CONCEPT
+[id="configuring-a-proxy-during-installation-ocm_{context}"]
+= Configuring a proxy during installation using {cluster-manager}
+
+If you are installing an {product-title} cluster into an existing Virtual Private Cloud (VPC), you can optionally enable a cluster-wide HTTP or HTTPS proxy during installation. You can enable a proxy only for clusters that are deployed by using the Customer Cloud Subscription (CCS) model.
```

Comment:
I think you can cut _optionally_ in this case. I think the context conveys that this is not a requirement.

Reply:
Good point, thank you. Resolved.